regression/cbmc-java/inherited_static_field6/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.class
+
+^VERIFICATION (SUCCESSFUL)|(FAILED)$
+--
+--
+This test can either pass or fail as the first 'x' reference may legitimately be guessed to belong to
```

Comment:
Is it not incorrect for this to be non-deterministic. Surely if x could below to both `parent` and `interface_without_static` it should be allowing cbmc the possibility that they belong to either or both or them?

Reply:
Will address the correctness question in the top level thread